### PR TITLE
alerting: update Slack configuration

### LINKED ISSUE #501: Several slack receiver parameters need some explanation
The Slack incoming webhook documentation linked to in the alertmanager configuration doesn't mention several of the parameters available in the `slack_configs` section:
- `color`
- `title`
- `title_link`
- `pretext`
- `fallback`

While `title` and `title_link` I can guess at (first line of text, emphasised, and associated link), the `color` parameter's defaults are non-obvious ("danger" and "good"...), and `pretext` and `fallback` I can only guess at.

### PR DESCRIPTION
**Pull request by simonpasquier**
Fixes #501.

### CODE REVIEW

File: content/docs/alerting/configuration.md [hunk 1]
```diff
@@ -449,20 +451,49 @@ Slack notifications are sent via [Slack webhooks](https://api.slack.com/incoming
...
+```
+
+### `<field_config>`
+
+The fields are documented in the [Slack API documentation](https://api.slack.com/docs/message-attachments#fields).
+
+```yaml
+title: <tmpl_string>
+value: <tmpl_string>
+[ short: <boolean> | default = slack_config.short_fields ]
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md]: slack_config.actions?

File: content/docs/alerting/configuration.md [hunk 2]
```diff
@@ -449,20 +451,49 @@ Slack notifications are sent via [Slack webhooks](https://api.slack.com/incoming
 channel: <tmpl_string>
 
 # API request data as defined by the Slack webhook API.
-[ color: <tmpl_string> | default = '{{ if eq .Status "firing" }}danger{{ else }}good{{ end }}' ]
+[ icon_emoji: <tmpl_string> ]
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md]: At this stage we should probably sort alphabetically

File: content/docs/alerting/configuration.md (hunk 1 above)
Comment (simonpasquier) [reply content/docs/alerting/configuration.md]: Hmm, I don't get your remark.

Comment (brian-brazil) [reply content/docs/alerting/configuration.md]: You had this at a different indentation previously

**Pull request merged**